syntax = "proto3";

message Number {
    float value = 1;
}

message Numbers {
	float number1 = 2;
	float number2 = 3;
}

service Calculator {
	rpc SquareRoot(Number) returns (Number) {}
	rpc Sum(Numbers) returns (Number) {}
	rpc Difference(Numbers) returns (Number) {}
	rpc Multiplication(Numbers) returns (Number) {}
	rpc Division(Numbers) returns (Number) {}
}
